tests/launchsecurity/sev.go

```diff
@@ -197,6 +187,31 @@ var _ = Describe("[sig-compute]AMD Secure Encrypted Virtualization (SEV)", decor
 		}
 	}
 
+	parseVirshInfo := func(info string, expectedKeys []string) map[string]string {
```

Comment:
This might be a good function to move to a shared files for utils. It can be done in a following PR

Reply:
Agree, this is a good candidate for common utils. Though, on the other hand, I think it makes sense to move it when there are several users of that function. Otherwise, the utils tend to grow.